### ISSUE TITLE
Using a relative -path for Import-VcCmApp causes errors

### ISSUE THREAD

**Issue by grantemsley** [aaronparker/Install-VisualCRedistributables#24]
# Expected Behavior

Not getting errors, and applications created properly.  Failing that, stopping before creating any applications.

# Current Behavior

Applications get created but with no deployment types and lots of errors.

# Failure Information (for bugs)

```
WARNING: Directory path '\2015\x64\Update3' does not appear to be a valid UNC path. Must be in the format of
\\myserver\share
Add-CMScriptDeploymentType :
At C:\Users\gemsley\Documents\WindowsPowerShell\Modules\VcRedist\Public\Import-VcCmApp.ps1:175 char:32
+                         $App | Add-CMScriptDeploymentType `
+                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (Microsoft.Confi...mentTypeCommand:AddScriptDeploymentTypeCommand) [Add
   -CMScriptDeploymentType], InvalidOperationException
    + FullyQualifiedErrorId : ContentValidationError,Microsoft.ConfigurationManagement.Cmdlets.AppMan.Commands.AddScri
   ptDeploymentTypeCommand

Add-CMScriptDeploymentType : Validation of input parameters failed. Cannot continue.
At C:\Users\gemsley\Documents\WindowsPowerShell\Modules\VcRedist\Public\Import-VcCmApp.ps1:175 char:32
+                         $App | Add-CMScriptDeploymentType `
+                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (Microsoft.Confi...mentTypeCommand:AddScriptDeploymentTypeCommand) [Add
   -CMScriptDeploymentType], InvalidOperationException
    + FullyQualifiedErrorId : ValidationFailed,Microsoft.ConfigurationManagement.Cmdlets.AppMan.Commands.AddScriptDepl
   oymentTypeCommand

Get-Item : Cannot find path '\\srv-configmgr\sourcefiles\apps\Microsoft\VCRedist' because it does not exist.
At C:\Users\gemsley\Documents\WindowsPowerShell\Modules\VcRedist\Public\Import-VcCmApp.ps1:152 char:32
+                 $Target = "$($(Get-Item -Path $CMPath).FullName)\$($Vc.Release)\ ...
+                                ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (\\srv-configmgr...rosoft\VCRedist:String) [Get-Item], ItemNotFoundExcep
   tion
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetItemCommand
```

This is repeated for each application.

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

This command causes the problem:
```
Import-VCCMApp -VCList $vclist -path . -cmpath "\\srv-configmgr\sourcefiles\apps\Microsoft\VCRedist" -smssitecode ABC -AppFolder VCRedist
```
This command does not:
```
Import-VCCMApp -VCList $vclist -path c:\users\gemsley\documents\temp -cmpath "\\srv-configmgr\sourcefiles\apps\Microsoft\VCRedist" -smssitecode ABC -AppFolder VCRedist
```

## Context

This seems to happen because UNC paths don't work while the current directory is in the configuration manager site provider.

Even simple commands like this fail:
```
cd c:\
dir \\server\path
<gets list of files>

cd ABC:\Application\VCRedist
dir \\server\path

dir : Cannot find path '\\server\path' because it does not exist.
At line:1 char:1
+ dir \\server\path
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (\\server\path:String) [Get-ChildItem], ItemNotFoundExcept
   ion
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
```

**Comment by aaronparker** [comment]
Have added private function `Get-ValidPath` and will update `Import-VcCmApp` to resolve full `-Path` when importing

**Comment by aaronparker** [comment]
Fixed in release release `v1.4.1.79`
